### PR TITLE
Add php deployment support

### PR DESCRIPTION
**Pull request by Benjiiim**
Now that kudu container on Azure App Service on Linux has PHP and composer support (cf Azure-App-Service/kudu#15), it seems a great time for kudu to have a php deployment script template.
PHP detection logic has been submitted on https://github.com/projectkudu/kudu/pull/2405
This PR was inspired from the Ruby support at #58 and #64

### CODE REVIEW

**Comment by davidebbo** [comment]
@nickwalkmsft can you help review this one?

**Comment by davidebbo** [comment]
@Hamster-Huey you may want to take a look as well since you did the Ruby changes.

File: lib/templates/deploy.bash.php.template [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# PHP Helpers
+# -----------
+
+initializeDeploymentConfig() {
+	if [ ! -e "$composerArgs" ]; then
```
Comment (nickwalkmsft) [inline lib/templates/deploy.bash.php.template]: Make $composerArgs all caps to match convention of other appsettings

**Comment by nickwalkmsft** [comment]
This looks good to me; David did you want to sign off on it at all?

File: lib/templates/deploy.bash.php.template (hunk 1 above)
Comment (Benjiiim) [inline lib/templates/deploy.bash.php.template]: Done

**Comment by davidebbo** [comment]
Looks harmless enough!

**Pull request merged**